aws/cf.go

```diff
@@ -48,6 +49,14 @@ func (s *Stack) TargetGroupARN() string {
 	return s.targetGroupARN
 }
 
+// IsComplete returns true if the stack status is a complete state.
+func (s *Stack) IsComplete() bool {
+	if s == nil {
+		return false
+	}
+	return isComplete(s.status)
```

Comment:
I think it would be better to move the whole isComplete(string) body to this function, because it's only used if you have a Stack and this will reduce number of functions spread.